ros_ign_bridge/test/resource/ign_subscriber.cpp.em

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <gtest/gtest.h>
+#include <ignition/transport.hh>
+
+#include <chrono>
+#include <memory>
+#include <string>
+
+#include "utils/test_utils.hpp"
+#include "utils/ign_test_msg.hpp"
+
+//////////////////////////////////////////////////
+/// \brief A class for testing Ignition Transport topic subscription.
+template<typename IGN_T>
+class MyTestClass
+{
+  /// \brief Class constructor.
```

Comment:
can you format the file ?

/// Doc
public/private: method name

Reply:
I didn't notice how weird the formatting was when I copied it over.  Fixed it.